runtime/src/lib.rs

```diff
@@ -524,6 +543,7 @@ construct_runtime!(
 		Anchor: anchor::{Module, Call, Storage},
 		Fees: fees::{Module, Call, Storage, Event<T>, Config<T>},
 		Nfts: nfts::{Module, Call, Event<T>},
+                MultiAccount: substrate_pallet_multi_account::{Module, Call, Storage, Event<T>},
```

Comment:
space ?

Reply:
I just realized that the substrate code base is using tabs as opposed to spaces(as most projects are).

Reply:
Yeah, we do not rustfmt to stay as close to Substrate (upstream) as possible to simplify upgrades. I hope they will rustfmt their codebase soon though ;)

Reply:
The paritytech/substrate repo is consistent with their [formatting](https://github.com/paritytech/substrate/blob/master/.editorconfig) which they are using tabs for all their rust code. I'll recreate a `rustfmt.toml` close equivalent of their configuration. I'll make a separate PR for formatting changes, after all the pending PR are merged, otherwise there will be a lot of formatting conflicts.

Reply:
Sounds good, thanks!